Added german translations

MadByteDE opened the pull request:
Translation by MadByteDE

Commit:
Added german translation (initial commit v1)

Commit:
Fixed typos and some translations

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+item.chisel.chisel_iron.name=Meißel
+item.chisel.chisel_diamond.name=Diamantmeißel
+item.chisel.chisel_hitech.name=iMeißel
```
I'd just keep it `iChisel` since it sounds more tech-y that way.

MadByteDE replied:
Sounds good to me. I'll change that.

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+item.chisel.chisel_iron.name=Meißel
+item.chisel.chisel_diamond.name=Diamantmeißel
+item.chisel.chisel_hitech.name=iMeißel
+item.chisel.offsettool.name=Ender-Versatzstab
```
I like that name.

MadByteDE replied:
Thanks

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+itemGroup.chiselCreativeTab=Meißel
+chisel.jei.title=Meißeln
+
+jei.chisel.desc.concrete_making=Füge nur etwas Wasser hinzu und schon kannst du deinen eigenen Beton herstellen!\n\nTauche Betonpulver in Wasser indem du es daneben plazierst oder lass' es ins Wasser fallen.
```
Needs a comma in front of `indem` and after `plazierst`.

MadByteDE replied:
Yep

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+item.chisel.chisel_hitech.name=iMeißel
+item.chisel.offsettool.name=Ender-Versatzstab
+
+chisel.spawnwither=Ein Meißel kann etwas gefährliches sein...
```
How about `Ein Meißel kann manchmal gefährlich sein...`?

MadByteDE replied:
English: `A Chisel can be a dangerous thing...`
Not sure about that. I would prefer my translation.

Vexatos replied:
Remember you do not always have to translate literally. If something makes more sense another way, or just sounds better, you're free to do it differently as long as the overall meaning stays the same.

If you prefer to keep your translation, "Gefährliches" will need to be capitalized.

tterrag1098 replied:
Hey, this feature was actually removed, so uh...maybe worry about something else? :stuck_out_tongue:

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+
+chisel.tooltip.blockname=%1$s (%2$s)
+
+tile.chisel.blockAluminum.name=Aluminumblock
```
Alumin*i*um.

MadByteDE replied:
yep :+1:

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.blockAluminum.name=Aluminumblock
+tile.chisel.blockAluminum.caution.desc.1=Vorsicht
+tile.chisel.blockAluminum.crate.desc.1=Versandkiste
+tile.chisel.blockAluminum.thermal.desc.1=Warm
```
...what? This makes no sense.

MadByteDE replied:
the thermal block in-game look like a heated block. I think something like "Warm" or "Erwärmt" or "Glühend" would be matching but "Thermal" sounds .. wrong in german.
I would like to have more ideas for that. What would be your translation?

Vexatos replied:
Hmm... I guess it is good enough the way it is.

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.andesite.bricks-cracked.desc.1=Gebrochene Ziegel
+tile.chisel.andesite.bricks-triple.desc.1=Breite Ziegel
+tile.chisel.andesite.bricks-encased.desc.1=Gefüllte Ziegel
+tile.chisel.andesite.braid.desc.1=Verflechtet
```
`Verflochten`.

MadByteDE replied:
yep

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.andesite.zag.desc.1=Zackig
+tile.chisel.andesite.circularct.desc.1=Kreisförmig
+tile.chisel.andesite.circularct.desc.2=Hat CTM
+tile.chisel.andesite.weaver.desc.1=keltisch
```
Capitalize.

MadByteDE replied:
done.

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.antiblock.green.desc.1=Grün
+tile.chisel.antiblock.brown.desc.1=Braun
+tile.chisel.antiblock.blue.desc.1=Blau
+tile.chisel.antiblock.purple.desc.1=Lila
```
More common word is `violett`.

MadByteDE replied:
Right, changed.

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.antiblock.brown.desc.1=Braun
+tile.chisel.antiblock.blue.desc.1=Blau
+tile.chisel.antiblock.purple.desc.1=Lila
+tile.chisel.antiblock.cyan.desc.1=Cyan
```
More common word is `türkis`. Should be more consistent with other mods.

MadByteDE replied:
Changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.antiblock.light_blue.desc.1=Hellblau
+tile.chisel.antiblock.magenta.desc.1=Magenta
+tile.chisel.antiblock.orange.desc.1=Orange
+tile.chisel.antiblock.white.desc.1=Weiss
```
Needs an ß there.

MadByteDE replied:
Changed.

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+
+tile.chisel.bloodMagic.name=Gemeißelte Blutrune
+tile.chisel.bloodMagic.bloodRuneArranged.desc.1=Angeordnet
+tile.chisel.bloodMagic.bloodRuneArranged.desc.2=Safe zur Dekoration
```
No good translation for "Safe"?

MadByteDE replied:
ups. 
Do you think it would be enough to translate it to "Zur Dekoration" instead of "Sicher zum/zur Dekoration" ? .. not sure.

Vexatos replied:
`Kann getrost dekorativ verwendet werden.`?
`Kann getrost als Dekoration verwendet werden.`?

MadByteDE replied:
Pretty long o.o I can do that but I'll have to test it in-game ... 
- I can't find this "Blood Rune" blocks ingame. Do they exist already?

Vexatos replied:
They're available with another mod, Blood Magic.

MadByteDE replied:
okay. I'll change it but considering that this will be displayed when hovering the block/item I would say that it's a bit to long. We'll see how it works out.

tterrag1098 replied:
You can add another description line if you need it

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.bloodMagic.name=Gemeißelte Blutrune
+tile.chisel.bloodMagic.bloodRuneArranged.desc.1=Angeordnet
+tile.chisel.bloodMagic.bloodRuneArranged.desc.2=Safe zur Dekoration
+tile.chisel.bloodMagic.bloodRuneArranged.desc.3=Nutzbar auf Blutaltaren
```
Doesn't `Verwendbar` sound better? Not sure.

MadByteDE replied:
yep, Ill change it .

Vexatos commented on src/main/resources/assets/chisel/lang/de_de.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.bookshelf_oak.rainbow.desc.1=Regenbogen
+tile.chisel.bookshelf_oak.necromancer-novice.desc.1=Geisterbeschwörer-Anfänger
+tile.chisel.bookshelf_oak.necromancer.desc.1=Geisterbeschwörer
+tile.chisel.bookshelf_oak.redtomes.desc.1=Rote Wälzer
```
:+1:

Vexatos reviewed (changes requested):
Please make sure to fix wrong words that appear multiple times (Like "Verflechtet" instead of "Verflochten") everywhere and not just where I pointed them out the first time.

Although I only had the time to review about the first 20% of the file, there is a lot of repitition in the rest, so the suggested changes should apply in a lot of other places (especially bricks and metal blocks).

Also, you need to fix the file name to `de_DE` instead of `de_de`.

Criticism on the suggestions is appreciated. I will read through the rest later.

MadByteDE commented:
Thank god there's someone reviewing it. I'll comment on the changes now.

Commit:
Fixed typos and translations 

Fixed stuff pointed out by Vexatos.

Commit:
Removed "de_de.lang" file...

now it's named as "de_DE.lang". 
Note: This will cause the translation to not work with Minecraft 1.11 -> has to be changed back for later Minecraft versions.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
"Lama", not "Llama".

MadByteDE replied:
changed.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
How about "Leerstein"? Would fit with other uses of "void" in Minecraft mods.

MadByteDE replied:
I thought Voidstone sounds good itself.. "Leerstein" doesn't sound that "cool" :D

Vexatos replied:
I like the word, it sounds... weird, but it is supposed to since the block is weird, too. And the entire concept of void in general.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
I'd just use "Alter Teppich" or something like that instead of translating "legacy" literally.

MadByteDE replied:
changed. Dunno why I used "Vermächnis-Teppich"... o.o

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Magenta` instead of `Magentia`.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Hellblaue Wolle`.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Magentafarbene Wolle`. Probably.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Pinke Wolle`.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Hellgraue Wolle`.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Uranblock`.... Please review all the metal names in general.

MadByteDE replied:
Changed to "Uranblock". 
Can't find other metal related problems.. but:
"Electrumblock" and "Kohlekoksblock" sound kinda weird, any suggestions or is it good as it is?

Vexatos replied:
"Electrum" is just "Elektrum" in German, coke is just ["Koks"](https://github.com/CovertJaguar/Railcraft-Localization/blob/767dfd17c536dedd5ad8208c8faeeb944bea80a9/assets/railcraft/lang/de_DE.lang#L837) or "Kokskohle", although I prefer the former.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
`Metal` is English, `Metall` is German... This occurs multiple times, please check your translations a bit more.

MadByteDE replied:
dunno why I translated this mod, should've be done by you beforehand ;)
- Found two mistakes in that case, changed.

Vexatos replied:
I don't have infinite time, unfortunately.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
How about `mittelgroß`, or something else more fitting?

MadByteDE replied:
Can't find the right line you refer to. Damn Github, what ya doing?!

Vexatos replied:
"Mittlere Fliesen" etc.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
There should not be any space in here or the line above.

MadByteDE replied:
Can't find the line.

Vexatos replied:
2574 and 2575, and a couple of similar lines. Maybe use hyphens there.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
You should not translate this since you cannot translate this.

If you do, do something completely unrelated, like `Es ist ein Wunderwerk des Sanitärhandwerks!`

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
This doesn't sound right.

MadByteDE replied:
Can't find the line you refer to.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
Needs a hyphen.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
This makes no sense. Should rather be `Ausblassystem` or something similar.

MadByteDE replied:
Not sure which word you refer to.. maybe "Ventilator" or "Abzug" ? If so then I don't see a problem with the translation.

Vexatos replied:
ExhaustPlating.

MadByteDE replied:
thanks, changed it.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
Be consistent with `verrostet`.

MadByteDE replied:
changed

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
No space here.

MadByteDE replied:
Please tell me the line you refer to.

Vexatos reviewed (changes requested):
I don't have the time to review everything today, You should look over the file once more and check for spaces in places where no space should be and places where hyphens should be, etc.

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.concrete_yellow.tiles-large.desc.1=Große Fliesen
+tile.chisel.concrete_yellow.tiles-small.desc.1=Kleine Fließen
+tile.chisel.concrete_yellow.chaotic-medium.desc.1=Ungeordnete Fließen
+tile
```
You have a lot of `Fließen` instead of `Fliesen`, too.

MadByteDE replied:
Changed

Vexatos commented:
Nice. Apparently github broke and now every comment on that review points at a line I didn't even mention.

MadByteDE commented:
Yeah, it's hard to fix that stuff when everything is in line 1247. :) 
btw I'm using the search and replace feature of my code editor to translate, no need to point out that I have to change stuff multiply times, but thanks :)

Commit:
Fixed some more stuff suggested  by Vexatos

Vexatos commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -23,7 +23,7 @@ chisel.tooltip.blockname=%1$s (%2$s)
 tile.chisel.blockAluminum.caution.desc.1=Vorsicht
 tile.chisel.blockAluminum.crate.desc.1=Versandkiste
-tile.chisel.blockAluminum.thermal.desc.1=Warm
+tile.chisel.blockAluminum.thermal.desc.1=Glühend
```
As I said, I'd have kept it "Warm". it is not supposed to look actually hot. The name comes from the textures being based on Thermal Foundation's metal blocks.

MadByteDE replied:
didn't knew that, but that said - wouldn't it be better to keep it as "Thermal" to slightly refer to the mod itself? "Warm" or "Glühend" does not make a difference unter these circumstances I think, it just sounds better in my opinion.
Other changes will go in later - some still need to be cleared up (because of that Github bug). Maybe It would be a good idea if you (Vexatos) maintance this translation further. Any chance that you change that stuff on your own?

Vexatos replied:
I really don't have much time to maintain translations of any more mods with big language files, unfortunately.

Commit:
Fixed more translations

Commit:
ups, fixed "Creeperstand" and "Mosaikstand" again

Commit:
Changed translation for ExhaustPlating to "Ausblassystem"

MadByteDE commented:
Revision needed.

btmnk commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+itemGroup.chiselCreativeTab=Meißel
+chisel.jei.title=Meißeln
+
+jei.chisel.desc.concrete_making=Füge nur etwas Wasser hinzu und schon kannst du deinen eigenen Beton herstellen!\n\nTauche Betonpulver in Wasser, indem du es daneben plazierst, oder lass' es ins Wasser fallen.
```
```suggestion
jei.chisel.desc.concrete_making=Füge nur etwas Wasser hinzu und schon kannst du deinen eigenen Beton herstellen!\n\nTauche Betonpulver in Wasser, indem du es daneben platzierst, oder lass' es ins Wasser fallen.
```

btmnk commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.carpet_magenta.name=Magentafarbende Wolle
+tile.chisel.carpet_magenta.legacy_magenta.desc.1=Alter Teppich
+tile.chisel.carpet_magenta.llama_magenta.desc.1=Lamateppich
+tile.chisel.carpet_lightblue.name=Helle Blaue Wolle
```
```suggestion
tile.chisel.carpet_lightblue.name=Hellblaue Wolle
```

MadByteDE replied:
Thanks, much better.

btmnk commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.carpet_gray.name=Graue Wolle
+tile.chisel.carpet_gray.legacy_gray.desc.1=Alter Teppich
+tile.chisel.carpet_gray.llama_gray.desc.1=Lamateppich
+tile.chisel.carpet_lightgray.name=Helle Graue Wolle
```
```suggestion
tile.chisel.carpet_lightgray.name=Hellgraue Wolle
```

MadByteDE replied:
yep

btmnk commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.carpet_cyan.name=Türkise Wolle
+tile.chisel.carpet_cyan.legacy_cyan.desc.1=Alter Teppich
+tile.chisel.carpet_cyan.llama_cyan.desc.1=Lamateppich
+tile.chisel.carpet_purple.name=Violettfarbende Wolle
```
```suggestion
tile.chisel.carpet_purple.name=Violette Wolle
```

btmnk replied:
Or "Violettfarbene"

btmnk replied:
There is the same mistake a few times further below.. So either just "farbene" or use the adjective without the "farbene". Usually works as well. 
(For Magenta it would also be just "Magenta Wolle" but since that sounds weird it's commonly changed to "Magentafarbene Wolle")

btmnk commented on src/main/resources/assets/chisel/lang/de_DE.lang:
```diff
@@ -0,0 +1,2774 @@
+tile.chisel.carpet_orange.name=Orange Wolle
+tile.chisel.carpet_orange.legacy_orange.desc.1=Alter Teppich
+tile.chisel.carpet_orange.llama_orange.desc.1=Lamateppich
+tile.chisel.carpet_magenta.name=Magentafarbende Wolle
```
```suggestion
tile.chisel.carpet_magenta.name=Magenta Wolle
```

Commit:
Update src/main/resources/assets/chisel/lang/de_DE.lang

Co-authored-by: Bitmonk <contact@bitmonk.de>

Commit:
Update src/main/resources/assets/chisel/lang/de_DE.lang

Co-authored-by: Bitmonk <contact@bitmonk.de>

Commit:
Update src/main/resources/assets/chisel/lang/de_DE.lang

Co-authored-by: Bitmonk <contact@bitmonk.de>

Commit:
Update src/main/resources/assets/chisel/lang/de_DE.lang

Co-authored-by: Bitmonk <contact@bitmonk.de>

Commit:
Update src/main/resources/assets/chisel/lang/de_DE.lang

Co-authored-by: Bitmonk <contact@bitmonk.de>

MadByteDE commented:
Seems like the mod is pretty much abandoned.. It's likely that changes won't be applied any time soon.

btmnk commented:
> Seems like the mod is pretty much abandoned.. It's likely that changes won't be applied any time soon.

Guess so..
Was curious what's going on here and just saw this PR and thought I could support - didn't even realize the last changes were years ago ^^"
I hope someone will come up with a worthy 1.16+ alternative :/

tterrag1098 commented:
This mod isn't abandoned, but the 1.10 version definitely is. If you want these merged it needs to be re-targeted at *at least* 1.12.

MadByteDE commented:
Good to know. Sorry for that, but since the last version released for MC 1.12, I assumed this is abandoned. I'll close this PR for now.
